CoreFoundation/String.subproj/CFString.c

```diff
@@ -5466,8 +5462,8 @@ void CFStringNormalize(CFMutableStringRef string, CFStringNormalizationForm theF
 
         contents = (uint8_t *)__CFStrContents(string) + __CFStrSkipAnyLengthByte(string);
 
-        for (;currentIndex < length;currentIndex++) {
-            if (contents[currentIndex] > 127) {
+        while (currentIndex < length;currentIndex++) {
+            if (contents[currentIndex++] > 127) {
```

Comment:
This is now double incremented? The change is not correct, or it is misleading to read.